docs/reference/r-requirements.md

```diff
@@ -7,4 +16,6 @@ For production environments, it is recommended to deploy at least 5 nodes for Zo
 - 12 storage devices
 - 10 GbE card
 
-The charm is meant to be deployed using `juju>=2.9.37`.
\ No newline at end of file
+## Supported architectures
+
+The charm is based on SNAP "[charmed-kafkal](https://snapcraft.io/charmed-kafka)", which is currently available for `amd64` only! The architecture `arm64` support is planned. Please [contact us](/t/13107) if you are interested in new architecture!
```

Comment:
**todo/question:** Is the `charmed-kafkal` a typo here?

Reply:
Yes! Nice! Great catch!